README.md

```diff
@@ -24,10 +24,11 @@ The [master](https://github.com/butterproject/butter-desktop) branch which conta
 
 #### Quickstart:
 
+
 1. `npm install -g grunt-cli bower`
-1. `npm install`
-1. `grunt build`
-1. `grunt start`
+2. `npm start`
```

Comment:
should be install, `npm start` is expected to start the app, i.e. do `npm install && grunt build && grunt start` i don't see that your PR implements that.